src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/common/DocumentSubtype.java

```diff
@@ -1,21 +1,9 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.model.common;
 
-import com.fasterxml.jackson.annotation.JsonValue;
+public class DocumentSubtype {
 
-public enum DocumentSubtype {
-    WILL("will"),
-    SSCS1("sscs1"),
-    COVERSHEET("coversheet");
+    public static final String WILL = "will";
+    public static final String SSCS1 = "sscs1";
+    public static final String COVERSHEET = "coversheet";
```

Comment:
These values would be `Will` `SSCS1` and `Coversheet`

Reply:
no change here

Reply:
I meant those will also be getting updated. Of course since supplier would be sending these values there won't be any change on our side but at some point this needs to be updated :)

Reply:
shouldn't the old mapping remain as is?

Reply:
They are remain till services have confirmed the dependency on these subtypes have been removed. https://tools.hmcts.net/confluence/display/RBS/Bulk+Scanning+Feature+Changes